docker-bits/6_remote-desktop.Dockerfile

```
@@ -340,7 +340,7 @@ RUN pip3 install --force websockify==0.9.0 \
     && tar -xf /tmp/novnc.tar.gz -C /tmp/ \
     && mv /tmp/noVNC-${NO_VNC_VERSION} /opt/novnc \
     && rm /tmp/novnc.tar.gz \
-    && chown -R $NB_UID:$NB_GID /opt/novnc
+    && chown -R $NB_UID:$NB_GID /opt/novnc 
```

Remove trailing whitespace
```suggestion
    && chown -R $NB_UID:$NB_GID /opt/novnc
```